File: CS/LocalizerExample/Form1.cs
using DevExpress.XtraSpreadsheet.Localization;
using System.Windows.Forms;

namespace LocalizerExample
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            XtraSpreadsheetLocalizer.Active = new CustomSpreadsheetLocalizer(); 
        }
    }
}

File: VB/LocalizerExample/Form1.vb
Imports DevExpress.XtraSpreadsheet.Localization
Imports System.Windows.Forms

Namespace LocalizerExample
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
            XtraSpreadsheetLocalizer.Active = New CustomSpreadsheetLocalizer()
        End Sub
    End Class
End Namespace
